hiss/application/migrations/0001_initial.py

```diff
@@ -28,15 +28,16 @@ class Migration(migrations.Migration):
                 ('classification', models.CharField(choices=[(None, '-- Select Option --'), ('Fr', 'Freshman'), ('So', 'Sophomore'), ('Jr', 'Junior'), ('Sr', 'Senior'), ('Ot', 'Other')], max_length=2, verbose_name='What classification are you?')),
                 ('grad_term', models.CharField(choices=[(None, '-- Select Option --'), ('Fall 2019', 'Fall 2019'), ('Spring 2020', 'Spring 2020'), ('Fall 2020', 'Fall 2020'), ('Spring 2021', 'Spring 2021'), ('Fall 2021', 'Fall 2021'), ('Spring 2022', 'Spring 2022'), ('Fall 2022', 'Fall 2022'), ('Spring 2023', 'Spring 2023'), ('Other', 'Other')], max_length=11, verbose_name='What is your anticipated graduation date?')),
                 ('num_hackathons_attended', models.CharField(choices=[(None, '-- Select Option --'), ('0', 'This will be my first!'), ('1-3', '1-3'), ('4-7', '4-7'), ('8-10', '8-10'), ('10+', '10+')], max_length=22, verbose_name='How many hackathons have you attended?')),
-                ('previous_attendant', models.BooleanField(choices=[(True, 'Yes'), (False, 'No')], default=False, verbose_name='Have you attended TAMUhack before?')),
+                ('previous_attendant', models.BooleanField(choices=[(True, 'Yes'), (False, 'No')], default=False, verbose_name='Have you attended HowdyHack before?')),
                 ('tamu_student', models.BooleanField(choices=[(True, 'Yes'), (False, 'No')], default=True, verbose_name='Are you a Texas A&M student?')),
                 ('extra_links', models.CharField(blank=True, max_length=200, verbose_name="Point us to anything you'd like us to look at while considering your application")),
                 ('question1', models.TextField(max_length=500, verbose_name='Tell us your best programming joke')),
                 ('question2', models.TextField(max_length=500, verbose_name="What is the one thing you'd build if you had unlimited resources?")),
-                ('question3', models.TextField(max_length=500, verbose_name="What is a cool prize you'd like to win at TAMUhack?")),
+                ('question3', models.TextField(max_length=500, verbose_name="What is a cool prize you'd like to win at HowdyHack?")),
```

Comment:
Question: Since the next upcoming hackathon is TAMUhack, shouldn't the question be "Have you attended TAMUhack before?" Or possibly ask if they've attended either HowdyHack or TAMUhack?

Reply:
That's a good point! This should be updated to use the [`EVENT_NAME`](https://github.com/tamuhack-org/Ouroboros/blob/188c6346b33e9f98ed38f797ad3b331331a61496/hiss/hiss/settings/customization.py#L2) setting.

Reply:
Update: The field actually DOES use the `EVENT_NAME` setting.